internal/controller/rbac/definition/roles.go

```diff
@@ -48,13 +48,15 @@ const (
 
 	valTrue = "true"
 
-	suffixStatus = "/status"
+	suffixStatus     = "/status"
+	suffixFinalizers = "/finalizers"
 )
 
 var (
-	verbsEdit   = []string{rbacv1.VerbAll}
-	verbsView   = []string{"get", "list", "watch"}
-	verbsBrowse = []string{"get", "list", "watch"}
+	verbsEdit           = []string{rbacv1.VerbAll}
+	verbsView           = []string{"get", "list", "watch"}
+	verbsBrowse         = []string{"get", "list", "watch"}
+	verbsOwnerReference = []string{"update"}
```

Comment:
```suggestion
	verbsUpdate = []string{"update"}
```

Please rename all instances of `verbsOwnerReference` to `verbsUpdate`. The naming convention is "verbs that allow X", not "verbs for X".